ql/src/test/queries/clientpositive/rename_translated_external_table.q

```diff
@@ -0,0 +1,18 @@
+set metastore.metadata.transformer.class=org.apache.hadoop.hive.metastore.MetastoreDefaultTransformer;
+set hive.fetch.task.conversion=none;
+set hive.compute.query.using.stats=false;
```

Comment:
These properties seem no use for this case?

Reply:
ack